src/pages/dashboard/DashboardPage.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import { useGroupBy } from 'src/api/groupByService'
+import { PageContainer } from '../components/PageContainer'
+import OperatorHbarChart from './OperatorHbarChart/OperatorHbarChart'
+import './DashboardPage.scss'
+
+const DashboardPage = () => {
+  const data = useGroupBy({
+    dateTo: '2023-01-10',
+    dateFrom: '2023-01-01',
```

Comment:
let's have this less hard coded, can have them use now +- a week or something

Reply:
done :)